apps/builddao/widget/components/modals/CreateProject.jsx

```diff
@@ -0,0 +1,304 @@
+      border-bottom-left-radius: 0.375rem;
+      border-bottom-right-radius: 0.375rem;
+      min-height: 9em;
+      * {
+        color: var(--text-color, #fff);
+      }
+    }
+    .nav-link {
+      color: var(--text-color, #fff);
+    }
+  }
+  .form-check {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-left: 0px;
+  }
+`;
+useEffect(() => {
+  console.log({
+    title,
+    description,
+    location,
+    contributors,
+    twitter,
+    gitHub,
+    telegram,
+    website,
+    selectedTabs,
+    tags,
+    avatar,
+    coverImage,
+  });
+}, [
+  title,
+  description,
+  location,
+  contributors,
+  twitter,
+  gitHub,
+  telegram,
+  website,
+  selectedTabs,
+  tags,
+  avatar,
+  coverImage,
+]);
+
+return (
+  <Modal
+    open={showModal}
+    title={"Create Project"}
+    onOpenChange={toggleModal}
+    toggle={toggle}
+  >
+    <Main>
+      <div className="lhs d-flex flex-column gap-4">
+        <InputField
+          key={"Project-Title"}
+          label={"Project Title"}
+          placeholder={"Enter Project Title"}
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+        />
+
+        <div className="form-group">
+          <label className="mb-1">Description</label>
+          <TextEditor
+            value={description}
+            onChange={(e) => setDescription(e.target.value)}
+          />
+        </div>
+        <InputField
+          key={"Location"}
+          label={"Location"}
+          placeholder={"Enter location"}
+          value={location}
+          onChange={(e) => setLocation(e.target.value)}
+        />
+        <div className="form-group">
+          <label className="mb-1">Project Contributors</label>
+          <Typeahead
+            multiple
+            options={
+              allMainnetAddresses ?? ["frank.near", "ellie.near", "jane.near"]
+            }
+            allowNew
+            placeholder="Enter Contributors username e.g frank.near, ellie.near"
+            selected={contributors}
+            onChange={(e) => handleContributors(e)}
+          />
+        </div>
+
+        <InputField
+          key={"twitter"}
+          label={"Twitter"}
+          placeholder={"twitter link"}
+          value={twitter}
+          onChange={(e) => setTwitter(e.target.value)}
+        />
+
+        <InputField
+          key={"github"}
+          label={"GitHub"}
+          placeholder={"github link"}
+          value={gitHub}
+          onChange={(e) => setGitHub(e.target.value)}
+        />
+
+        <InputField
+          key={"telegram"}
+          label={"Telegram"}
+          placeholder={"telegram link"}
+          value={telegram}
+          onChange={(e) => setTelegram(e.target.value)}
+        />
+
+        <InputField
+          key={"website"}
+          label={"Website"}
+          placeholder={"telegram link"}
+          value={website}
+          onChange={(e) => setWebsite(e.target.value)}
```

Comment:
can we add checks for URLs

Reply:
Actually, I'll check just for the website then change socials to just take the handles or usernames